core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerProxy.scala

```diff
@@ -161,7 +162,7 @@ class ContainerProxy(
 
     // cold start (no container to reuse or available stem cell container)
     case Event(job: Run, _) =>
-      implicit val transid = job.msg.transid
+      implicit val transId: TransactionId = job.msg.transid
```

Comment:
Small nit but we use sid and transid consistently throughout (no camel case) so I wouldn’t make this change (unless it’s done everywhere).